python/cudf/cudf/tests/test_list.py

```diff
@@ -819,3 +819,38 @@ def test_memory_usage():
     assert s1.memory_usage() == 44
     s2 = cudf.Series([[[[1, 2]]], [[[3, 4]]]])
     assert s2.memory_usage() == 68
+
+
+@pytest.mark.parametrize(
+    "data, idx, expected",
+    [
+        (
+            [[{"f2": {"a": 100}, "f1": "a"}, {"f1": "sf12", "f2": None}]],
+            0,
+            [{"f1": "a", "f2": {"a": 100}}, {"f1": "sf12", "f2": None}],
+        ),
+        (
+            [
+                [
+                    {"f2": {"a": 100, "c": 90, "f2": 10}, "f1": "a"},
+                    {"f1": "sf12", "f2": None},
+                ]
+            ],
+            0,
+            [
+                {"f1": "a", "f2": {"a": 100, "c": 90, "f2": 10}},
+                {"f1": "sf12", "f2": None},
+            ],
+        ),
+        (
+            [[[[1, 2]], [[2], [3]]], [[[2]]], [[[3]]]],
+            0,
+            [[[1, 2]], [[2], [3]]],
+        ),
+        ([[[[1, 2]], [[2], [3]]], [[[2]]], [[[3]]]], 2, [[[3]]]),
+    ],
+)
+def test_nested_list_extract_host_scalars(data, idx, expected):
+    series = cudf.Series(data)
+
+    assert series[idx] == expected
```

Comment:
We don't need to define `expected` in the parameters, we can just pull it directly from `data[idx]`. Also we should test multiple values of `idx`, not just `0`.
```suggestion
    assert series[idx] == data[idx]
```

Reply:
Done